nengo_loihi/hardware/builder.py

```diff
@@ -435,6 +436,15 @@ def collect_axons(n2core, core, block, axon, cx_ids):
             assert (n_blocks == 0
                     or (n_blocks == 1 and block is core.blocks[0]))
             assert len(block.probes) == 0
+            tchip_id_source = n2board.n2Chips[
+                core.board.chip_index(core.chip)].id
```

Comment:
Is there any easier way to do this? i.e., a way that doesn't involve around 4 levels of indirection?

Reply:
Nope. You could add an `index` property to `chip` that returns `self.board.chip_index(self)` to simplify that middle part.

Reply:
Done